src/tests/functional/base_func_tests/src/base/low_precision_transformations/layer_transformation.cpp

```diff
@@ -10,6 +10,8 @@
 #include "openvino/util/common_util.hpp"
 
 namespace LayerTestsUtils {
+using namespace ov::test;
```

Comment:
Is this actually used? I can see that you use `ov::test::` directly below

Reply:
Initially, It was needed to be able to use `operator<<` for `InputShape`. I removed redundant `ov::test::` usages in this file, thanks